contracts/src/libraries/FROST.sol

```diff
@@ -3,54 +3,123 @@ pragma solidity ^0.8.30;
 
 import {Secp256k1} from "@/libraries/Secp256k1.sol";
 
-/// @title FROST
-/// @notice Implementation of FROST(secp256k1, SHA-256) cyphersuite.
+/**
+ * @title FROST
+ * @notice Implementation of the FROST(secp256k1, SHA-256) ciphersuite.
+ */
 library FROST {
     using Secp256k1 for Secp256k1.Point;
 
+    // ============================================================
+    // TYPES AND STRUCTS
+    // ============================================================
+
+    /**
+     * @notice Type alias for a FROST participant identifier.
+     */
     type Identifier is uint256;
 
+    /**
+     * @notice Group commitment used in FROST signing.
+     * @param identifier The participant identifier associated with this commitment.
+     * @param d The first public nonce commitment point.
```

Comment:
it is not the first and second, but rather a hidding and binding nonce commitment (see https://datatracker.ietf.org/doc/html/draft-irtf-cfrg-frost-08#section-5.1). 

Ref: https://github.com/safe-research/shieldnet/blob/main/validator/src/consensus/signing/nonces.ts#L11-L20